catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -293,28 +298,76 @@ void registryInfoQuery() throws UnsupportedQueryException {
         SourceResponse identityMetacard = query(queryRequest);
         if (identityMetacard.getResults()
                 .size() > 0) {
-            remoteName = identityMetacard.getResults()
+            String metacardTitle = identityMetacard.getResults()
                     .get(0)
                     .getMetacard()
                     .getTitle();
+            setId(identityMetacard.getResults()
```

Comment:
Just use the metacardTitle string here instead of pulling it out of the metacard.